Editor/StatusBox.cs

```diff
@@ -39,15 +39,15 @@ public StatusBox()
             var asset = Resources.Load<VisualTreeAsset>("EditorWindow/Components/StatusBox");
             asset.CloneTree(this);
 
-            _container = this.Q("StatusBox");
-            _statusTextLabel = _container.Q<Label>("StatusBoxLabel");
+            _statusTextLabel = this.Q<Label>("StatusBoxLabel");
             _externalButton = this.Q<Button>("StatusBoxExternalButton");
             _externalButtonLabel = this.Q<Label>("StatusBoxExternalButtonLabel");
-            _elementLocalizer = new ElementLocalizer(_container);
-            this.Q<Button>("StatusBoxCloseButton").RegisterCallback<ClickEvent>(_ =>
-            {
-                Close();
-            });
+            _elementLocalizer = new ElementLocalizer(this);
+            this.Q<Button>("StatusBoxCloseButton").RegisterCallback<ClickEvent>(_ => { Close(); });
+
+            AddToClassList("status-box");
+            AddToClassList("separator");
+            AddToClassList("separator--horizontal");
```

Comment:
This change seems a bit weird to me. Why can't it be structured the way it was before and added to the top of the uxml?